modules/ROOT/pages/troubleshoot-nfs.adoc

```diff
@@ -33,6 +33,10 @@ The error message `Unable to access file system at [NFS SERVER]: rpc mount expor
 If the mount fails, you are probably missing some NFS-required libraries. If this happens, install them by running: +
 `yum install nfs-utils nfs-utils-lib`
 
+=== "mount failed: Not Permitted"
+
+The error message `mount failed: exit status 32. Mounting command ... not permitted` may occurr when trying to mount with the `noresvport` option since the ports may be limited. If this happens, remove the `noresvport` option and add the `insecure` flag to the mounting command.
```

Comment:
```suggestion
The error message `mount failed: exit status 32. Mounting command ... not permitted` might occur when trying to mount using the `noresvport` option because the ports might be limited. If this happens, remove the `noresvport` option and add the `insecure` flag to the mount command.
```

Reply:
I always use "might" (possibility) rather than "may" (permission). Also, I think "since" implies time rather than causation.

Reply:
excellent, thanks!